i17obot/handlers.py

```diff
@@ -30,6 +28,39 @@ async def start(message: types.Message):
     )
 
 
+async def language(message: types.Message):
+    await types.ChatActions.typing()
+    keyboard_markup = types.InlineKeyboardMarkup()
+    for code, language in config.AVAILABLE_LANGUAGES.items():
+        keyboard_markup.row(types.InlineKeyboardButton(language, callback_data=code))
+
+    await bot.send_message(
+        message.chat.id,
+        "Translate from English to which language?",
+        disable_web_page_preview=True,
+        parse_mode="markdown",
+        reply_markup=keyboard_markup,
+    )
+
+
+async def set_language(query: types.CallbackQuery):
+    await types.ChatActions.typing()
+    await update_user(query.from_user.id, language_code=query.data)
+
+    # TODO: Find a better way to translate the messages the bot sends
+    if query.data == "es":
+        template = "Idioma elegido: *{}*.\nUse el comando /language para cambiar."
```

Comment:
@gilgamezh would you be able to review this string? This was based on Google Translate :)

Reply:
That's perfect. I would change use for "Utilice" 

`Idioma elegido: *{}*.\nUtilice el comando /language para cambiarlo.`